<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DltDbus::Form</class>
 <widget class="QWidget" name="DltDbus::Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>DBus</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QTextBrowser" name="textBrowserDBus"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Header</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QTextBrowser" name="textBrowserHeader"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>Payload</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QTextBrowser" name="textBrowserPayload"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Payload Hex</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QTextBrowser" name="textBrowserPayloadHex"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
